docs/pages/setup/guides/terraform-provider.mdx

```diff
@@ -27,6 +27,7 @@ Create a folder `teleport-terraform` to hold some temporary files:
 
 ```bash
 mkdir -p teleport-terraform
+cd teleport-terraform
```

Comment:
I think this is great to encapsulate all getting started and guide files in a folder. Makes it easier to remove afterward.